vmss.tf

```diff
@@ -73,7 +74,100 @@ resource "azurerm_proximity_placement_group" "ppg" {
   tags                = merge(var.tags_map, {"weka_cluster": var.cluster_name})
 }
 
-resource "azurerm_linux_virtual_machine_scale_set" "vmss" {
+data "azurerm_shared_image_version" "shared_image_version" {
```

Comment:
eventually you just need the imager id, so this should be the parameter, no need for all the variables and no need for this data resource

Reply:
I am using shared gallery want custom image to be public so i need **shared_image_version** and you are right I can use id but I dont like it,` ` **"id": "/Subscriptions/d2f248b9-d054-477f-b7e8-413921532c2a/Providers/Microsoft.Compute/Locations/eastus/Publishers/zscaler1579058425289",`**`

Reply:
but it is better then 4 params in a map in tfvars ..

Reply:
OK I will change it